### PR TITLE
Frequency: Increase width of graph and remove commented code fix #2295

### LINKED ISSUE #2295: Frequency Spectrum: Cleanup code, make content larger, reduce whitespace
The Frequency Spectrum Goodie has now been converted to use templates, but the size of the content has been scaled down and now has extra whitespace.

It would be great if we could reduce the whitespace to make the content large so it's easier to read. If necessary we can add some CSS to expand the size of the graph, but it would be ideal to keep the visuals inside the existing box.

There are also many lines of commented code we should remove.

------ 

IA Page:  https://duck.co/ia/view/frequency_spectrum

### PR DESCRIPTION
**Pull request by Sayanc93**
fixes #2295

IA: https://duck.co/ia/view/frequency

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by moollaza** [comment]
@Sayanc93 please make sure you mention the issue this PR fixes in the PR description, not the title. That way the issue will automatically be closed when the PR is merged.

File: share/goodie/frequency_spectrum/frequency_spectrum.css [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .zci--conversions {
-    width: 90%;
+    width: 100%;
```
Comment (moollaza) [inline share/goodie/frequency_spectrum/frequency_spectrum.css]: I'm not sure the fix is so simple -- I imagine there's still a large whitespace on the left and the font size is still quite small? 

We probably need to modify the SVG

**Comment by Sayanc93** [comment]
@moollaza  why is this goodie behaves differently in duckpan server and in production? check the images below -
1) production - 
screen shot 2016-02-27 at 2 29 26 pm
2) development - 
screen shot 2016-02-27 at 2 35 33 pm

Now as of my solution, I tried my change on duckduckgo.com and it gave me something like this - 
screen shot 2016-02-27 at 2 38 58 pm

which is bigger and clearer...but this change doesn't reflect on development...I dont understand whats going on

**Comment by moollaza** [comment]
@Sayanc93 the UI in production is the old code that was recently changed. That was the result of the Goodie returning pure HTML instead of using templates which we wanted to replace.

With templates the full width of the IA will only be as wides as the IA Container allows (which is the same width as the link results). So 100% width doesn't make it much larger, but typically IA's should be wider than the IA container.

After the next release the IA in production will match the UI you're seeing in development.

**Comment by moollaza** [comment]
@Sayanc93 Ping! I'm just checking in to see if you're still interested in finishing this one up? We'd love to see this go live!

There's absolutely no rush, just wanted to make sure you're not stuck on anything.

**Comment by Sayanc93** [comment]
@moollaza - my bad...i was having trouble with codio last time i tried to login..so gave up..but finally got access to it today. Is it possible to setup local environment for DuckDuckHack?
also updated my other PRs...i'll work on this later tonight :)

**Comment by mintsoft** [comment]
@Sayanc93 You can use a VM or a local install; the instructions are here: https://github.com/duckduckgo/p5-app-duckpan/blob/master/README.md

However it will probably take quite a long time to download and build the dependencies required.

**Comment by moollaza** [comment]
@Sayanc93 Ping! I'm just checking in to see if you're still interested in finishing this one up? Do you still need help getting your dev environment setup? Are you able to use Codio?

**Comment by sahildua2305** [comment]
@Sayanc93 Please note that you have tagged the wrong IA link in the PR. Please correct that so that this PR is linked to the correct IA. Thanks! :smile:

**Pull request closed without merging**